### ISSUE TITLE
error compiling : No such file or directory - which fontforge

### ISSUE THREAD

**Issue by desubmarine** [FontCustom/fontcustom#189]
helo,
when i try to execute : font custom watch, I get the following error.
I've installed ruby and fontforge.
error compiling : No such file or directory - which fontforge
C:/Ruby200-x64/../lib/fontcustom/base.rb:
33:in ``': No such file or directory - which fontforge (Errno::ENOENT)
        from C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/fontcustom-1.3.3/lib/fontcu
stom/base.rb:33:in`check_fontforge'
        from C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/fontcustom-1.3.3/lib/fontcu
stom/base.rb:8:in `initialize'

anyone could help?

thx
juna

**Comment by blingerson** [comment]
I have the same error message on windows as well.

Here is my version info:

fontcustom-1.3.3

Copyright (c) 2000-2014 by George Williams. See AUTHORS for Contributors.
 License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
 with many parts BSD <http://fontforge.org/license.html>. Please read LICENSE.
 Based on sources from 09:31 AWST  1-May-2015-ML-TtfDb-D.
 Based on source from git with hash: 1aba9cbdf300dc7dcb40eba5e2f5c31f40a37818
no xdefs_filename!
TESTING: getPixmapDir:C:/Program Files/FontForgeBuilds/share/fontforge/pixmaps
TESTING: getShareDir:C:/Program Files/FontForgeBuilds/share/fontforge
TESTING: GResourceProgramDir:C:/Program Files/FontForgeBuilds/bin
trying default theme:C:/Program Files/FontForgeBuilds/share/fontforge/pixmaps/resources
fontforge 09:31 AWST  1-May-2015
libfontforge 20150501

**Comment by jdell64** [comment]
I believe my latest PR should address this.

**Comment by julmot** [comment]
Same issue

**Comment by jdell64** [comment]
Oh, silly me... space issue.

**Comment by julmot** [comment]
Yes it is. But I have installed FontForge to C:\FontForgeBuilds like described in the docs. So there's nothing more I can do besides of following the docs. Very painful.

**Comment by julmot** [comment]
![2017-07-06 16_47_04-cmd](https://user-images.githubusercontent.com/8310677/27916918-c5eb7550-626a-11e7-8298-036b2612e4db.png)
